Document/0x05b-Basic-Security_Testing.md

```diff
@@ -142,7 +142,7 @@ You'll come across different use cases on how you can use adb commands when test
 
 [apktool](https://ibotpeaches.github.io/Apktool/) is used to unpack Android app packages (APKs). Simply unzipping APKs with the standard `unzip` utility leaves some files unreadable. `AndroidManifest.xml` is encoded into binary XML format which isn’t readable with a text editor. Also, the app resources are still packaged into a single archive file.
 
-When run with default command line flags, apktool automatically decodes the Manifest file to text-based XML format and extracts the file resources (it also disassembles the .DEX files to smali code – a feature that we’ll revisit later in this book).
+When run with default command line flags, apktool automatically decodes the app manifest file to text-based XML format and extracts the file resources (it also disassembles the .DEX files to smali code – a feature that we’ll revisit later in this book).
```

Comment:
note: you are using app manifest and android manifest interleaved. best is to stick to an `app its manifest` and shorten it or to the Android Manifest of the app, but app manifest is kind of confusing ... to me...

Reply:
I agree :) This is changed now across files to Android Manifest, it should be clearer now.